docs/pages/versions/unversioned/sdk/background-task.mdx

```diff
@@ -0,0 +1,208 @@
+
+### When are background tasks run?
+
+The Expo Background Task API leverages each platform to execute tasks at the most optimal time for both the user and the device.
+
+This means that the task may not run immediately after it is scheduled, but it will run at some point in the future if the system decides so.
+
+A background task will only run if the battery has enough charge (or the device is plugged into power) and the network is available. Without these conditions, the task won't execute. The exact behavior will vary depending on the operating system.
+
+### When will they stop?
+
+The native platform APIs will stop running background tasks if the user explicitly kills the app (they will be resumed when the user restarts the app) or if the app is uninstalled. If the device is rebooted or the app is killed by the system, the background tasks will be resumed.
+
+> **Info** The non-deterministic behavior of background tasks is intentional, aimed at optimizing battery life and enhancing user experience. If a user force stops an app, the system interprets this as a signal not to run background tasks for that app until the user restarts it. On iOS an app is force stopped if the user swipes up on the app in the app switcher. On Android this behaviour depends on the device vendor. Some vendors have implementations where the app is force stopped when the user removes the app from the recent apps list.
+
+## Platform differences
+
+### Android&ensp;<PlatformTags platforms={['android']} />
+
+On Android, the [`WorkManager`](https://developer.android.com/topic/libraries/architecture/workmanager) API allows specifying a minimum interval for a task to run (at least 15 minutes). The task will execute sometime after the interval has passed, provided the specified conditions are met.
+
+### iOS&ensp;<PlatformTags platforms={['ios']} />
+
+On iOS, the [`BGTaskScheduler`](https://developer.apple.com/documentation/backgroundtasks/bgtaskscheduler) API decides the best time to launch your background task. The system will consider the battery level, the network availability, and the user's usage patterns to determine when to run the task.
+
+## Known limitations
+
+### iOS&ensp;<PlatformTags platforms={['ios']} />
+
+The [`Background Tasks`](https://developer.apple.com/documentation/backgroundtasks) API on iOS is not available on simulators, it is only available when running on a physical device.
+
+## Installation
+
+<APIInstallSection />
+
+## Configuration&ensp;<PlatformTags platforms={['ios']} />
+
+To be able to run background tasks on iOS, you need to add the `processing` value to the `UIBackgroundModes` array in your app's **Info.plist** file. This is required for background fetch to work properly.
+
+**If you're using [CNG](/workflow/continuous-native-generation/)**, the required `UIBackgroundModes` configuration will be applied automatically by prebuild.
```

Comment:
Can we make this part the first paragraph in this section and then the current first paragraph (`you need to add the `processing`) can follow with the code snippet so we are not repeating the information? Right now, we are repeating the part both in the first paragraph and in the collapsible about adding `processing` value manually.